docs/reference/resp2-to-resp3-replies.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "RESP2 to RESP3 replies migration guide"
+linkTitle: "RESP2 to RESP3 replies migration guide"
```

Comment:
This needs to be shorted as it appears as the title in the left-hand navigation tree.